src/integration.test.ts

```diff
@@ -1,6 +1,7 @@
-import { keyPairFromSeed, SkynetClient } from "./index";
+import { SkynetClient } from "./client";
+import { pki } from "node-forge";
 
-const { publicKey, privateKey } = keyPairFromSeed("test");
+const { publicKey, privateKey } = pki.ed25519.generateKeyPair();
```

Comment:
this is not what I meant, you should still use `keyPairFromSeed` but the seed you pass should be random instead of `"test"`